content/en/tracing/guide/ingestion_sampling_use_cases.md

```diff
@@ -0,0 +1,155 @@
+
+Ingestion mechanisms are configurations within the Datadog Agent and Datadog tracing libraries. If you are using OpenTelemetry SDKs to instrument your applications, read [Ingestion Sampling with OpenTelemetry][2].
+
+This guide helps you understand when and how to use ingestion control configurations depending on the main use cases you might encounter. It covers:
+
+- [Determining which ingestion mechanisms are used](#determining-which-ingestion-mechanisms-are-used) for a given service
+- [Use cases that focus on keeping particular types of traces](#keeping-certain-types-of-traces)
+- [Use cases that focus on reducing ingested traces](#reducing-ingestion-for-high-volume-services)
+
+
+## Determining which ingestion mechanisms are used
+
+To identify which ingestion mechanisms are currently used in your Datadog environment, navigate to the [Ingestion Control Page][3].
+
+{{< img src="/tracing/guide/ingestion_sampling_use_cases/ingestion_control_page.png" alt="Ingestion Control Page" style="width:90%;" >}}
+
+The table gives insights on ingested volumes *by service*. The Configuration column provides a first indication of the current set up. It shows: 
+- `AUTOMATIC` if the sampling rate calculated in the Datadog Agent is applied to the traces that start from the service. Read more about the specifics of [Datadog Agent ingestion logic][5].
+- `CONFIGURED` if a custom sampling rate configured in the tracing library is applied to the traces that start from the service.
+
+Click on services to see details about what sampling decision makers (for example Agent or tracing library, rules or sample rates) are used for each service, as well as what [ingestion sampling mechanisms][1] are leveraged for ingested spans' services. 
+
+{{< img src="/tracing/guide/ingestion_sampling_use_cases/service-ingestion-summary.png" alt="Service Ingestion Summary" style="width:90%;" >}}
+
+## Keeping certain types of traces
+
+### Keeping entire transaction traces
+
+Ingesting entire transaction traces ensures visibility over the **end-to-end service request flow** for specific individual requests.
+
+#### Solution: Head-based sampling
+
+Complete traces can be ingested with [head-based sampling][4] mechanisms: the decision to keep or drop the trace is determined from the first span of the trace, the *head*, when the trace is created. This decision is propagated through the request context to downstream services.
+
+{{< img src="/tracing/guide/ingestion_sampling_use_cases/head_based_sampling_keep.png" alt="Head-based Sampling" style="width:100%;" >}}
+
+To decide which traces to keep and drop, the Datadog Agent computes default sampling rates for each service to apply at trace creation, based on the application traffic:
```

Comment:
It would be useful to add a link back to this section about **DD_APM_MAX_TPS**: https://docs.datadoghq.com/tracing/trace_pipeline/ingestion_mechanisms/?tab=java#in-the-agent .

Reply:
Added a link